packages/squiggle-lang/src/fr/scale.ts

```diff
@@ -115,10 +138,16 @@ export const library = [
             min: min ?? undefined,
             max: max ?? undefined,
             tickFormat: tickFormat ?? undefined,
-            exponent,
+            exponent: exponent || SqPowerScale.defaultExponent,
```

Comment:
I figure we might as well have these all be optional

Reply:
Same question about zero or negative exponent value as in symlog comment above.